flake.nix

```diff
@@ -32,11 +32,16 @@
         in
         {
           default = with pkgs; mkShell {
+            strictDeps = true;
```

Comment:
Is this a hidden undocumented feature? I dont understand what this does

Reply:
Do you see the OPENSSL variables? They were leaking through from your home environment
Not undocumented, but read the manual for `mkDerivation`